docs/next.config.js

```diff
@@ -39,12 +42,4 @@ const config = {
   },
 }
 
-if (process.env.NODE_ENV === 'development') {
```

Comment:
Will this still work with symlinks?

Reply:
Can make it work! Only have the playground set up that way now, but can get the docs site going too.